modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2AuthorizationCodeGrantJWTTokenTestCase.java

```diff
@@ -323,10 +324,7 @@ public void testValidateAdditionalUserClaims() {
         applicationConfig.getRequestedClaimList().forEach(claim -> {
             if (authorizingUser.getUserClaims().get(claim) != null) {
                 assertNotNull(accessTokenClaims.getClaim(claim.getOidcClaimUri()),
-                        "Claim " + claim.getOidcClaimUri() + " not found in the access token.");
-                assertEquals(accessTokenClaims.getClaim(claim.getOidcClaimUri()),
-                        authorizingUser.getUserClaims().get(claim),
-                        "Value for claim " + claim.getOidcClaimUri() + " is incorrect in the access token.");
+                        "User claim " + claim.getOidcClaimUri() + " not found in the access token.");
```

Comment:
@mpmadhavig why assert equals is removed ?

Reply:
This has been missed during e refactoring effort. Fixed with 52ebe10566c95461912b7fa85477396ca386a1d5